README-dev.md

```diff
@@ -66,57 +91,46 @@ The wrappers are built with the plugin. `dvp build` has a hidden `--dev` flag. T
 vsdk_root = /path/to/vsdk_repo_root
 ```
 
-### Versioning
-The first thing to do is to change the version number. There are _two_ places where the version needs to be changed unfortunately. For almost all cases, this will simply involve incrementing the "build number", which is the three-digit number at the very end of the version.
-
-1. In the root of the SDK codebase, open the `build.gradle` file, and change the version.
-2. In `tools/src/main/python/dlpx/virtualization/_internal/settings.cfg` change the `package_version` property.
-
-The two versions should be identical.
-
-This repository is going through a transition in which Gradle will eventually be removed. The version string will eventually only live in the individual packages themselves.
-
-### Testing
-
-Currently, there are three types of SDK testing: unit, manual, and blackbox.
+## Testing
 
-#### Unit Testing
+Currently, there are three types of SDK testing: unit, manual, and functional (blackbox).
 
-Running `./gradlew test` from the top level of the repository will run all SDK unit tests. Smaller sets of tests can be run from inside each directory (`common`, `platform`, etc.) by going into that directory and running `../gradlew test`. These can also be run through your IDE.
+### Unit Testing
 
-#### Testing sdk-gate changes with app-gate code
+Go into one of the package directories (common, dvp, libs, platform, tools) and run the following commands (if you haven't done it already):
+1. Install the package's development dependencies: `pip install -r requirements.txt`.
+2. Install the package itself in editable mode: `pip install -e .`.
+3. Run unit tests: `python -m pytest src/main/python`.
 
-At the moment blackbox refers to a property file in the app-gate to determine the version of the SDK to install for tests so this property always needs to updated for automated testing.
+Currently, there's no way to locally run unit tests in all packages with one command. However, soon, there will be a Github Action which will run all pre-commit checks on a pull request. 
```

Comment:
nit: The accuracy is great, but I don't think we need to be this accurate. The pre-commit run is coming in a couple days and we will probably forget to update this. Just saying:
```
There is no way to run the unit tests in all packages locally with one command. However, they will be ran automatically through GitHub Actions when you open a pull request.
```

Doesn't have to be that exactly.

Reply:
I changed the readme as you suggested.